README.md

```diff
@@ -117,326 +68,113 @@ https://pmi.pay1.de/
-
-### Paypal Sandbox Accounts
-
-To test with Paypal, you need own Sandbox Buyer credentials via a developer account. Available from commercetools, too; please contact support. 
-
-## Contribute Improvements
-
-If you want to add a useful functionality or found a bug please open an issue here to announce and discuss what you
-have in mind.  Then fork the project somewhere or in GitHub and create a pull request here once you're done. 
-
-## Development Notes
-
-Please bear in mind that this repository should be free of any IDE specific files, configurations or code. Also, the use
- of frameworks and libraries should be transparent and reasonable.
-
-## Create a custom version
-
-Just fork it. The MIT License allows you to do anything with the code, commercially or noncommercial. 
-Contributing an Improvement is the better Idea though because you will save maintanance work when not forking. 
-
-## Appendix 1: Shell script template that sets the environment variables
-
-(fill in the values required for your environment)
-
-```
-#!/bin/sh
-export CT_PROJECT_KEY=""
-export CT_CLIENT_ID=""
-export CT_CLIENT_SECRET=""
-export CT_START_FROM_SCRATCH="false"
-
-export PAYONE_KEY=""
-export PAYONE_MERCHANT_ID=""
-export PAYONE_MODE=""
-export PAYONE_PORTAL_ID=""
-export PAYONE_SUBACC_ID=""
-
-# from here on only test related
-
-export CT_PAYONE_INTEGRATION_URL=""
-
-export TEST_DATA_VISA_CREDIT_CARD_NO_3DS=""
-export TEST_DATA_VISA_CREDIT_CARD_3DS=""
-export TEST_DATA_3_DS_PASSWORD=""
-export TEST_DATA_SW_BANK_TRANSFER_IBAN=""
-export TEST_DATA_SW_BANK_TRANSFER_BIC=""
-```
-
-## Appendix 2: Alternative configuration via properties file
-
-Instead of the shell script described above a Java properties file called `gradle.properties` can be put in the project
-root directory to configure the build/test. It will be picked up by Gradle (and is ignored by Git).
-
-```
-# use the Gradle daemon, i.e. re-use the JVM for builds
-org.gradle.daemon=true
-
-# integration service configuration
-
-CT_PROJECT_KEY=<commercetools project key>
-CT_CLIENT_ID=<commercetools client ID>
-CT_CLIENT_SECRET=<commercetools client secret>
-
-PAYONE_KEY=<PAYONE Key>
-PAYONE_MERCHANT_ID=<PAYONE merchant ID>
-PAYONE_MODE=<PAYONE mode (live or test)>
-PAYONE_PORTAL_ID=<PAYONE portal ID>
-PAYONE_SUBACC_ID=<PAYONE subaccount>
-
-# test configuration
-
-CT_PAYONE_INTEGRATION_URL=<URL of the integration service instance under test>
-
-TEST_DATA_VISA_CREDIT_CARD_NO_3DS=<see PAYONE Test data documentation>
-TEST_DATA_VISA_CREDIT_CARD_3DS=<see PAYONE Test data documentation>
-TEST_DATA_3_DS_PASSWORD=<see PAYONE Test data documentation>
-TEST_DATA_SW_BANK_TRANSFER_IBAN=<see PAYONE Test data documentation>
-TEST_DATA_SW_BANK_TRANSFER_BIC=<see PAYONE Test data documentation>
-```
+The job is processing every tenant payments sequentially one-by-one in order they are described in `TENANTS` property,
```

Comment:
Does this feature worth to invest more time/human resources + computer resources to support multithreading there? If yes - let's create a separate issue task and implement, but i don't think it worths...